recipe_scrapers/lekkerensimpel.py

```diff
@@ -29,10 +32,39 @@ def image(self):
         return image["content"] if image else None
 
     def ingredients(self):
-        return self.schema.ingredients()
+        if self.schema.ingredients():
+            return self.schema.ingredients()
+
+        ingredient_header = self.soup.find("strong", string="Benodigdheden:")
+        if ingredient_header:
+            return self.process_ingredients(
+                ingredient_header.parent.parent.find_next("ul")
+            )
+
+        ingredient_header = self.soup.find("p", string="Benodigdheden:")
+        if ingredient_header:
+            return self.process_ingredients(ingredient_header.parent.find_next("ul"))
+
+        raise ElementNotFoundInHtml("Could not find ingredients.")
+
+    def process_ingredients(self, container):
+        ingredients = container.findChildren("li")
+
+        return [normalize_string(i.get_text()) for i in ingredients]
```

Comment:
Optional: I'd suggest not re-using this code between the two old-format ingredient scraping approaches.  Allowing the historic-version-scrapers to diverge more easily if they need to (in other words: not sharing code between each of the old-format page scrapers) would seem to make sense to me.  My reasoning is something to do with (human) workflow scalability, that I can try to do a better job of describing if you're curious.

Reply:
Do you have an opinion on this @jaapio?  This is the only item I'd like to check; apart from that I think this is ready to merge.

Reply:
I used this same approach in the keukenliefdenl parser, so that's why I did it this way, there is no specific reasoning behind this, execpt for the fact that I do not like duplication, and I try to hide complexity when reading the code. Which seems to be the opposite from your approach? 

Right now the scraping ingredients method reads for me like this:

- Do search for element x 
- if found process element x
- Do search for element y 
- if found process element y
- otherwise error! 

If I understand you correctly you want something more like:
- Do search for element x 
- if found process element x
- find all children
- normalize children
- Do search for element y 
- if found process element y
- find all children
- normalize children
- otherwise error! 

Where the first approach is easier to understand, as far as I understand the human brain. The cognitive load it smaller as some details are hidden. Unless you dive deeper. 

But in the end both ways are good, and most likely, I will never touch this particular scraper again, unless I find a bug or another format that's not covered. Where you might want to refactor and maintain this for longer. So your opinion is more important to me, than my way of thinking.

Reply:
Yes, one of the benefits but also responsibilities for any maintainer is to decide what kind of code is acceptable -- but that also means trying to reduce the amount of effort required to adjust and fix the code in (not-always-predictable) future.

I suppose the question is: do the two pretzels sit in separate small plates on a bed of mustard, or is there a single rectangular plate with a single shared mustard base?

If I predict that one of the pretzels might be served separately for some reason, then it's going to be slightly less messy (presentation-wise) to achieve that if they are on separate (albeit adjacent) plates to begin with.  However, that could also look quite strange.  Why two plates for these two pretzels?

I have no idea, really.  I think I'd prefer to remove the shared method in case the code will be divided out into separate scrapers in future.  And also vaguely based on the dont-repeat-yourself rule of three -- duplicating code so that it appears twice can be OK, but at three times that's usually a sign to introduce shared functions.

I'm comfortable making the change here since I'm figuratively plating the pretzels.

Reply:
:)  you're welcome - thank you!

Reply:
Done, please let me know if any additional changes are required.

Reply:
Thank you :+1:

Looks great, and ready to merge.  I've applied one nitpick fix, and will merge this soon.